src/lib/utils/shopify.js

```diff
@@ -137,7 +137,7 @@ export async function getProduct(id) {
               currencyCode
             }
           }
-          variants(first: 100) {
+          variants(first: 250) {
```

Comment:
Why is increasing this necessary? Can we just remove this limit?

Reply:
Didn't seem worth it to open a separate PR, but I work with the Shopify API a lot and know that the maximum number of connections you can fetch for most graphql requests is 250, before you have to do pagination. So the 100 limit seemed kind of arbitrary and you might as well use the maximum.

You can't remove this or you will get an error.

Reply:
Moved to #45